pyensembl/sequence_data.py

```diff
@@ -36,22 +38,29 @@ class SequenceData(object):
     """
     def __init__(
             self,
-            fasta_path,
+            fasta_paths,
             require_ensembl_ids=False,
             cache_directory_path=None):
-        self.fasta_path = abspath(fasta_path)
-        self.fasta_directory_path, self.fasta_filename = split(self.fasta_path)
+
+        if isinstance(fasta_paths, string_types):
+            fasta_paths = [fasta_paths]
+
+        self.fasta_paths = [abspath(path) for path in fasta_paths]
+        self.fasta_directory_paths = [split(path)[0] for path in self.fasta_paths]
+        self.fasta_filenames = [split(path)[1] for path in self.fasta_paths]
         if cache_directory_path:
-            self.cache_directory_path = cache_directory_path
+            self.cache_directory_paths = [cache_directory_path]*len(self.fasta_paths)
```

Comment:
I think the only two alternative solutions are to automatically create the cache directory in global cache directory supplied from datacache python package (e.g. `/Users/charlesmurphy/Library/Caches/pyensembl` on my Mac) or force the user to always specify `cache_directory_paths` when working with their own custom FASTA files. If neither solution sounds good, then keep the current solution.